content/pages/burials-and-memorials/application/530.md

```diff
@@ -4,12 +4,13 @@ entryname: burials
 layout: page-react.html
 ---
 <div id="main">
-  <nav class="va-nav-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" role="menubar" aria-label="Primary">
+  <nav aria-label="Breadcrumb" aria-live="polite" aria-relevant="additions text" class="va-nav-breadcrumbs js-visual"
+  id="va-breadcrumbs">
+    <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/burials-and-memorials/"> Burials and Memorials </a></li>
       <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/"> Survivors and Dependents </a></li>
-      <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
+      <li><a aria-current="page" href="/burials-and-memorials/application/530/">Handling Burial Costs (Allowances)</a></li>
```

Comment:
I think the hierarchy is 
```
Home > Burials and Memorials > Survivors and Dependents > Handling Burial Costs (Allowances) > Apply for burial benefits
```

So, I think we should be adding an item here instead of changing an existing one?